container/incremental_load.sh.tpl

```diff
@@ -201,7 +201,7 @@ EOF
 
   # On macOS, clean all xattrs from the files we're going to load.
   if [ "$(uname)" == "Darwin" ]; then
-    echo "Cleaning xattrs from files on macOS..."
+    echo "Cleaning xattrs from files on macOS..." >&2
```

Comment:
@kevinqian-db this log line actually broke macOS as I explained on Slack since it polluted the output
I'm just logging it to stderr to fix it